devtools-ci-index.yaml

```diff
@@ -2134,6 +2165,24 @@
             timeout: '30m'
         - '{ci_project}-{git_repo}-fabric8-analytics-pylint':
             git_repo: fabric8-analytics-server
+        - '{ci_project}-{git_repo}-fabric8-hdd':
+            git_organization: fabric8-hdd
+            git_repo: supervisor
+            ci_project: 'devtools'
+            ci_cmd: '/bin/bash cico_run_tests.sh'
```

Comment:
Before we send these changes to almighty/almighty-jobs , we need to make sure that the cico(test, build, eslint) build scripts are done as well

Reply:
Yes, They are merged